spec/index.bs

```diff
@@ -1243,6 +1243,12 @@ The default is used if a value is not explicitly specified in {{GPUDeviceDescrip
         The maximum value of the product of the `workgroup_size` values for a
         compute stage {{GPUShaderModule}} entry-point.
 
+    <tr><td><dfn>maxComputeWorkgroupSize</dfn>
+        <td>{{GPUExtent3D}} <td>Per-component Higher <td>[256, 256, 64]
+    <tr class=row-continuation><td colspan=4>
+        The maximum value of the product of the `workgroup_size` values for a
```

Comment:
Does "of the product" implies the product operation, i.e. multiplication here? It looks like there should be none of it.

Reply:
```suggestion
        The maximum values of the `workgroup_size` values for a
```